example/static/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>landline-api test page</title>
+    <!-- I'm assuming here that Promises and XHR2 are avilable. Most notable, IE doesn't yet support promises, although they're currently in the technical preview. -->
```

Comment:
That's also true (and something I was mulling over when I posted that original comment). I think in the spirit of minimal test cases, _maybe_ it'll be useful at some point. But totally agreed, it'll probably never be a priority.